api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -379,7 +379,7 @@ class StocksQueryModel(BaseModel):
     price_category_id: int | None
     order_by: offers_repository.StocksOrderedBy = offers_repository.StocksOrderedBy.BEGINNING_DATETIME
     order_by_desc: bool = False
-    page: int = 1
+    page: int = Field(1, gt=0)
```

Comment:
[détail / question de goût] 

```suggestion
    page: int = Field(1, ge=1)
```

... demande un peu moins de gymnastique de mon neurone. "La valeur par défaut est 1, parce qu'on accepte les valeurs >= 1" me semble (un tout petit) plus évident que "La valeur par défaut est 1, parce qu'on accepte les valeurs > 0".